kalamuna/kalastatic
# #165: Fix the deployment conditions

#165 The conditions were incorrect. This fixes them.

### QA

1. Visit https://dashboard.getpantheon.com/sites/99097056-c6bd-451e-a94b-fc8f7666fbe5#multidev/overview/git-branches
2. See that there is a `testbuild` git branch on Pantheon
3. Merge this PR
